src/main/java/frc/robot/Robot.java

```diff
@@ -39,6 +41,7 @@ public Robot()
     drivetrain = new Drivetrain();
     intake = new Intake();
     shooter = new Shooter();
+    driver = new Driver(this);
```

Comment:
Might have to import the Driver file, since it is not being recognized, causing the build to fail.